.ci/conditional-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+annotation=$(git tag -l --format='%(contents)' "$CIRRUS_TAG")
+last_commit_message=$(git log -1 --pretty=%B)
+
+if [[ "$annotation" == "$last_commit_message" ]]; then
+        echo "Not an annotaged-tag build. No need to release."
+        exit 0
+fi
+
+# problem with the date command: release at midnight may lead to errors
+files_to_upload=(
+    "$(date +%Y-%m-%d)-VAST-$(git describe)-Linux-Release.tar.gz"
```

Comment:
Just remove the date, annotated tags are unique.

Sidenote: Use `date --rfc-3339=date` instead in the future.